<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialMonitor</class>
 <widget class="QMainWindow" name="SerialMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialMonitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="serialConsole">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SerialPort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serialPortInfoListBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>BaudRate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="baudRateBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SerialInput</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="SerialInputConsole">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>219</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendpushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>New Line</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerialInterface">
    <property name="title">
     <string>SerialInterface </string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuSerialInterface"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
